airsonic-main/src/main/java/org/airsonic/player/service/search/DocumentFactory.java

```diff
@@ -57,7 +57,7 @@
     static {
 
         TYPE_ID = new FieldType();
-        TYPE_ID.setIndexOptions(IndexOptions.NONE);
+        TYPE_ID.setIndexOptions(IndexOptions.DOCS);
```

Comment:
The document design may be different from the legacy, but it is a mandatory modification.
An Index is required to successfully retrieve the primary key.
In other words, this modification prevents double registration.